web/web/api/v1/user.py

```diff
@@ -73,11 +73,11 @@ def modify_user(user_id):
 
     except MultipleResultsFound:
         arw.add_errors({user_id: 'Multiple results found for the given user id.'})
-        return arw.to_json()
+        return arw.to_json(None, 400)
     
     except NoResultFound:
         arw.add_errors({user_id: 'No results found for the given user id.'})
-        return arw.to_json()
+        return arw.to_json(None, 400)
```

Comment:
It isn't necessary right now, but since `errors` is populated, `data` won't be used, so you could just use the named param `status_code=400`. No problems though